laravel/app/Console/Kernel.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Console;
+
+use App\Console\Commands\StatsCleanup;
+use App\Console\Commands\StatsDownload;
+use Illuminate\Console\Scheduling\Schedule;
+use Illuminate\Foundation\Console\Kernel as ConsoleKernel;
+use Illuminate\Support\Facades\Artisan;
+
+class Kernel extends ConsoleKernel
+{
+    /**
+     * The Artisan commands provided by your application.
+     *
+     * @var array
+     */
+    protected $commands = [
```

Comment:
Define the class names of the commands that will be available for use from the scheduler